config/lib/seed.js

```diff
@@ -39,16 +54,8 @@ if (process.env.NODE_ENV === 'production') {
   //Add Local User
   User.find({username: 'user'}).remove(function () {
     var password = crypto.randomBytes(64).toString('hex').slice(1, 20);
-    var user = new User({
-      username: 'user',
-      password: password,
-      provider: 'local',
-      email: 'user@localhost.com',
-      firstName: 'User',
-      lastName: 'Local',
-      displayName: 'User Local',
-      roles: ['user']
-    });
+    seedUser.password = password;
+    var user = new User(seedUser);
```

Comment:
I didn't make that change, that's part of the original code for the seeding.
IIRC it is removing any existing user and adding a new one so you can gain access to it, and it's only happening when you're on dev/test.